apps/backoffice-v2/src/common/components/organisms/Header/Header.Navbar.tsx

```diff
@@ -15,9 +16,7 @@ import { CheckSquare } from 'lucide-react';
  */
 export const Navbar: FunctionComponent = () => {
   const { data: filters } = useFiltersQuery();
-  const search = useSearch({
-    strict: false,
-  });
+  const [searchParams] = useSearchParamsByEntity();
```

Comment:
`useSearchParamsByEntity` picks a different zod schema depending on the `entity` search query param.